code/src/java/pcgen/io/exporttoken/BonusToken.java

```diff
@@ -46,9 +46,7 @@ public String getTokenName()
 		return TOKENNAME;
 	}
 
-	/**
-	 * @see pcgen.io.exporttoken.Token#getToken(java.lang.String, pcgen.core.PlayerCharacter, pcgen.io.ExportHandler)
-	 *
+	/*
```

Comment:
```suggestion
	/**
```
Keep the JavaDoc format.